website/content/api-docs/secret/transit.mdx

```diff
@@ -214,12 +220,23 @@ provided AES key. The wrapped AES key should be the first 512 bytes of the
 ciphertext, and the encrypted key material should be the remaining bytes.
 See the BYOK section of the [Transit secrets engine documentation](/docs/secrets/transit#bring-your-own-key-byok)
 for more information on constructing the ciphertext.
+// TODO: Update text
```

Comment:
@Gabrielopesantos Can you review this TODO?

Reply:
Hello tjperry
I didn't mean to commit this file as it is still in development. I will intend to fix these TODOs and add additional information for the final review. Sorry for the notification.